go/chat/server.go

```diff
@@ -2286,3 +2286,59 @@ func (h *Server) GetStaticConfig(ctx context.Context) (res chat1.StaticConfig, e
 		DeletableByDeleteHistory: chat1.DeletableMessageTypesByDeleteHistory(),
 	}, nil
 }
+
+func (h *Server) ResolveUnfurlPrompt(ctx context.Context, arg chat1.ResolveUnfurlPromptArg) (err error) {
+	var identBreaks []keybase1.TLFIdentifyFailure
+	defer func() {
+		// squash all errors coming out of here
+		err = nil
+	}()
+	ctx = Context(ctx, h.G(), arg.IdentifyBehavior, &identBreaks, h.identNotifier)
+	defer h.Trace(ctx, func() error { return err }, "ResolveUnfurlPrompt")()
+	uid, err := utils.AssertLoggedInUID(ctx, h.G())
+	if err != nil {
+		return err
+	}
+	atyp, err := arg.Result.ActionType()
+	if err != nil {
+		return err
+	}
+	fetchAndUnfurl := func() error {
```

Comment:
nit: move this definition up so `atyp` is defined right before we do the `switch`